apps/css-workshop/src/components/Button.astro

```diff
@@ -0,0 +1,22 @@
+---
+type Props = {
+  variant?: 'high-visibility' | 'borderless' | 'cta';
+  size?: 'small' | 'large';
+  active?: boolean;
+  shift?: 'left' | 'right';
```

Comment:
The `shift` prop probably shouldn't exist within `Button`. For one, we don't have this prop in React. But more importantly, it only makes sense for borderless icon buttons. I think we might want to create a separate `IconButton` astro component, similar to React

Reply:
Okay, will add `IconButton` component.